### PR TITLE
feat: add ckb indexer example

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: example/src/main/java/org/nervos/ckb/service/RpcService.java [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (duanyytop) [inline example/src/main/java/org/nervos/ckb/service/RpcService.java:1]: Is this an empty file?

File: example/src/main/java/org/nervos/ckb/service/RpcCallback.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package org.nervos.ckb.service;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
+interface RpcCallback<T> {
```
Comment (duanyytop) [inline example/src/main/java/org/nervos/ckb/service/RpcCallback.java:4]: I think `RpcCallback` in SDK can be reused.

File: example/src/main/java/org/nervos/ckb/service/RpcResponse.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package org.nervos.ckb.service;
+
+/** Copyright © 2020 Nervos Foundation. All rights reserved. */
+public class RpcResponse<T> {
```
Comment (duanyytop) [inline example/src/main/java/org/nervos/ckb/service/RpcResponse.java:4]: I think `RpcResponse ` in SDK can be reused.

File: example/src/main/java/org/nervos/ckb/service/RpcCallback.java (hunk 2 above)
Comment (shaojunda) [reply example/src/main/java/org/nervos/ckb/service/RpcCallback.java:4]: Yes, you are right I will fix this.

Comment (shaojunda) [reply example/src/main/java/org/nervos/ckb/service/RpcCallback.java]: updated: [`fdb0bd9` (#348)](https://github.com/nervosnetwork/ckb-sdk-java/pull/348/commits/fdb0bd92701d7591e8cd118768f22856de06df45)

File: example/src/main/java/org/nervos/ckb/service/CkbIndexerApi.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.Collections;
+import org.nervos.ckb.indexer.CkbIndexerCellResponse;
+import org.nervos.ckb.indexer.CkbIndexerCellsCapacityResponse;
+import org.nervos.ckb.indexer.SearchKey;
+import org.nervos.ckb.utils.Numeric;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
```
Comment (duanyytop) [inline example/src/main/java/org/nervos/ckb/service/CkbIndexerApi.java:12]:
```suggestion
/** Copyright © 2020 Nervos Foundation. All rights reserved. */
```

**Review by duanyytop — APPROVED** [review]

**Pull request merged**